microcosm_fastapi/conventions/schemas.py

```diff
@@ -33,11 +68,18 @@ def dict(self, *args, exclude_none=True, **kwargs):
 
 
 def SearchSchema(item_class):
-    class _SearchSchema(BaseModel):
+    class _SearchSchema(EnhancedBaseModel):
         links: Optional[LinksSchema] = Field(alias="_links")
         count: int
         items: List[item_class]
+        offset: int
+        limit: int
+
+        __config__ = item_class.__config__
 
-    _SearchSchema.__name__ = "Search" + item_class.__name__
+    # We append "List" to end to remain backwards compatible
+    # If we wanted to change to the following schema: "Search" + item_class.__name__`
+    # then we could create two duplicate our models...
```

Comment:
Not sure what this comment means; can you elaborate?

Reply:
Let me clarify: What I'm trying to say is that before we had the SchemaName defined as:
```
_SearchSchema.__name__ = "Search" + item_class.__name__
```
But I've changed it to 
```
_SearchSchema.__name__ = item_class.__name__ + "List"
```
So that we remain in sync with what we've done in the past (i.e this is the current naming convention we currently have inside of our current swagger specs -> https://github.com/globality-corp/marquez/blob/master/marquez/swagger/babilonia.v2.json#L110). I think when I wrote the comment I thought that we might be breaking places like where we do the `get_model` syntax (e.g see [here](https://github.com/globality-corp/babilonia/blob/09bf2e987bc781fd8a5a7286312587c850f90a77/babilonia/tests/daemon/qna_suggestions_daemon/classifiers/test_service_offering_classifier.py#L53)) and we might have been specifying a model defined like:
```
self.babilonia_v2.get_model("XXXX..List")
```

And if we defined our schema with the convention `Search..XXXX` then this would potentially break stuff. However, after doing a quick regex of our code base I couldn't actually see anywhere which utilises the `XXXX..List` models.

Hope that helps to explain where I was coming from on this one. By changing the format to "XXXX..List" we cause less errors in the api-contract-test as the current v2 babilonia schema and the converted v2 babilonia are more aligned. 

I think this comment actually makes more sense out of the code in a PR comment.